pkgs/by-name/vl/vlc-bittorrent/package.nix

```diff
@@ -0,0 +1,66 @@
+{ autoconf
+, autoconf-archive
+, automake
+, boost
+, fetchFromGitHub
+, gcc
+, gnumake
+, lib
+, libtool
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorent}";
+
+stdenv.mkDerivation rec {
+  pname = "vlc-bittorrent";
+  version = "2.15.0";
+
+  src = fetchFromGitHub {
+    owner = "johang";
+    repo = pname;
+    rev = "6810d479e6c1f64046d3b30efe78774b49d1c95b";
+    hash = "sha256-7FHeQYHbMKZJ3yeHqxTTAUwghTje+gEX8gSEJzfG5sQ=";
+  };
+
+  nativeBuildInputs = [
+    gcc
+    autoconf
+    autoconf-archive
+    automake
+    gnumake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libtorrent-rasterbar
+    libvlc
+    libtool
+    openssl
+    boost
+  ];
+
+  buildPhase = ''
+    autoreconf -i
+    ./configure --prefix=$out
+    make
+  '';
```

Comment:
Use [`autoreconfHook`](https://nixos.org/manual/nixpkgs/unstable/#setup-hook-autoconf) instead.

Reply:
You should also remove most `autoconf`-related packages from `nativeBuildInputs` when replacing them by `autoreconfHook` (I think `autoconf-archive` needs to stay though).

Reply:
Indeed, it does not build without `autoconf-archive`.